pom.xml

```diff
@@ -90,6 +92,7 @@
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>git-client</artifactId>
+      <version>4.0.0-rc3300.80c23f8e08ec</version>
```

Comment:
FWIW I like to add such things to `dependencyManagement` with a `TODO` comment to delete once `bom` picks it up.